Cloud release notes july 2024 (#6347)
umbraco/UmbracoDocs: umbraco-cloud/release-notes/2024-07-releasenotes.md

# Release Notes, July 2024

## [Sustainability Dashboard](https://docs.umbraco.com/umbraco-cloud/getting-started/the-umbraco-cloud-portal/sustainability-dashboard)

> We're excited to announce the launch of the Sustainability Dashboard in Umbraco Cloud. Its a powerful new tool designed to help you monitor and optimize the environmental impact of your cloud projects. As sustainability becomes increasingly important, we're committed to providing tools that empower you to make informed decisions about your digital infrastructure.

Reviewer: ```suggestion
We are excited to announce the launch of the Sustainability Dashboard in Umbraco Cloud. It is a powerful new tool designed to help you monitor and optimize the environmental impact of your cloud projects. As sustainability becomes increasingly important, we are committed to providing tools that empower you to make informed decisions about your digital infrastructure.
```

![Sustainability Dashboard](images/sustainability-dashboard.png)

Use it to track and analyze the carbon emissions associated with your Umbraco Cloud projects. The dashboard provides clear metrics, helping you understand the environmental impact of your cloud usage.

> The Sustainability Dashboard is available now to all Umbraco Cloud users. it is part of our ongoing commitment to supporting greener, more sustainable cloud solutions. We’re excited to see how it helps you in your journey towards a more eco-friendly digital presence.

Reviewer: ```suggestion
The Sustainability Dashboard is available now to all Umbraco Cloud users. It is part of our ongoing commitment to supporting greener, more sustainable cloud solutions. We are excited to see how it helps you on your journey towards a more eco-friendly digital presence.
```
